src/transformers/modeling_attn_mask_utils.py

```diff
@@ -0,0 +1,211 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import List, Optional, Tuple, Union
+
+import torch
+
+
+class AttnMaskConverter:
```

Comment:
Could it eventually be renamed to `AttentionMaskConverter` ? :innocent: 

(personal preference, but would rather have no abbreviations if we can have the full word without too much hassle)

Reply:
+1